.github/workflows/release_apk.yml

```diff
@@ -0,0 +1,31 @@
+name: Test_and_build_artifacts_on_release
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build_apk_aab:
+    name: Build release artifacts
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Build release APK and AAB after test
+        run: |
+          ./gradlew test
+          ./gradlew assembleRelease
+          ./gradlew bundleRelease
+      - name: Upload APK
+        uses: actions/upload-artifact@v2
+        with:
+          name: app-release.apk
```

Comment:
Is it ok to have this name or we need to use our final name for the app?

Reply:
I've renamed it but only in artifact file's name. To rename generated app I have to raise separate PR because there will be huge list of renaming